Makefile

```diff
@@ -49,25 +52,40 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 
 .PHONY: fmt
 fmt: ## Run go fmt against code.
-	go fmt ./...
+	$(GO_CMD) fmt ./...
+
+fmt-verify:
+	@out=`$(GO_FMT) -w -l -d $$(find . -name '*.go')`; \
```

Comment:
Does `-d` only print the diff or actually modify the files? I think it only prints the diff, so we don't need to check the status of the repo in verify.sh

Reply:
```
  -d    display diffs instead of rewriting files
```
It will print the diff, but the exit code will still be 0.

the small check on verify.sh is to get an exit code, so we can use the script during prow testing

Reply:
Yeah, I get that. What I'm saying is that we don't need to run `git diff` at the end, because `make fmt-verify` would make the script fail already.